docs/Documentation/eagle_to_kestrel_transition.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Transitioning from Eagle to Kestrel
+---
+
+## Overview of steps
+
+Eagle is set to be decommissioned at the end of March, 2024. This page is meant to provide all necessary information to transition a project from Eagle to Kestrel. Transitioning a project can be broken down into [NUMBER] steps:
+
+1. Getting access to Kestrel
+2. Moving your files from Eagle to Kestrel
+3. Understanding the options for running your software on Kestrel
+
+    a. How to check if your software is available as a module on Kestrel
+
+    b. How to build your own software on Kestrel
+
+    c. General environment recommendations
+
+4. Submitting your jobs on Kestrel
+
+If you find yourself stuck on any of the above steps, please reach out to hpc-help@nrel.gov as soon as possible.
+
+## 1. Getting access to Kestrel
+
+Access to Kestrel is nearly identical to access to Eagle. For example, you can log into Kestrel by opening a terminal and ssh'ing to the machine:
+
+`ssh [your username]@kestrel.hpc.nrel.gov`
+
+The exact process for accessing the machine is different for NREL employees than non-NREL employees. If you are having difficulty logging on, please see [this page](/Documentation/Systems/Kestrel/#accessing-kestrel) for more detailed information on logging onto Kestrel.
+
+Once you've successfully logged onto Kestrel, you should next check that your project directory exists and that you have access to the project directory. 
+
+To do this, try `cd /projects/[your allocation name]`. If this directory does not exist, then check that you have been given a Kestrel allocation in Lex. If yes, then reach out to hpc-help@nrel.gov for assistance.
```

Comment:
If a user doesn't have a Kestrel allocation (or isn't a member of a Kestrel allocation), they won't be able to login to Kestrel at all. So, I think the recommendation to check Lex could either be removed (as a large majority of users do have an allocation) or moved up as a step to check if they are unable to login. I think this is more of an issue with the other systems where directories aren't automatically provisioned, or only a subset of users have access.